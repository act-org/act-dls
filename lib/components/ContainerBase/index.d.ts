/**
 * @prettier
 */
import * as React from 'react';
import { ContainerProps } from '@material-ui/core';
export declare type Props = ContainerProps;
declare const ContainerBase: React.FC<Props>;
export default ContainerBase;
//# sourceMappingURL=index.d.ts.map