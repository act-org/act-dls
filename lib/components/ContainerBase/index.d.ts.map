{"version":3,"file":"index.d.ts","sourceRoot":"","sources":["../../../src/components/ContainerBase/index.tsx"],"names":[],"mappings":"AAAA;;GAEG;AAEH,OAAO,KAAK,KAAK,MAAM,OAAO,CAAC;AAC/B,OAAO,EAAa,cAAc,EAAE,MAAM,mBAAmB,CAAC;AAE9D,oBAAY,KAAK,GAAG,cAAc,CAAC;AAEnC,QAAA,MAAM,aAAa,EAAE,KAAK,CAAC,EAAE,CAAC,KAAK,CAEmB,CAAC;AAEvD,eAAe,aAAa,CAAC"}