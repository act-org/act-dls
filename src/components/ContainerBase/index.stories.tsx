/**
 * @prettier
 */

import * as React from 'react';

import ContainerBase from '.';

export const Base = (): React.ReactElement<any> => (
  <ContainerBase>Container</ContainerBase>
);

export default {
  component: ContainerBase,
  parameters: {
    info: {
      text: 'This is the base variant of the Container component.',
    },
  },
  title: 'Atoms/Container',
};
