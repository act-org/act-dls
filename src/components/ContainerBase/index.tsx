/**
 * @prettier
 */

import * as React from 'react';
import { Container, ContainerProps } from '@material-ui/core';

export type Props = ContainerProps;

const ContainerBase: React.FC<Props> = (
  props: Props,
): React.ReactElement<any> => <Container {...props} />;

export default ContainerBase;
