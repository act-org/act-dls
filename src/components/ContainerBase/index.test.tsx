/**
 * @prettier
 */

import * as React from 'react';

import render from '~/helpers/test/render';

import ContainerBase from '.';

describe('ContainerBase', () => {
  it('matches the snapshot', () => {
    const { container } = render(
      <ContainerBase maxWidth="lg">This is a test.</ContainerBase>,
    );

    expect(container).toMatchSnapshot();
  });
});
