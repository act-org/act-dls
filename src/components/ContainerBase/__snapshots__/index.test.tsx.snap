// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContainerBase matches the snapshot 1`] = `
<div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg"
  >
    This is a test.
  </div>
</div>
`;
